server/tests/functional/test_login.py

```diff
@@ -0,0 +1,20 @@
+def test_login(client):
+    """
+    GIVEN a Flask app
+    WHEN a user logs in
+    AND a POST request is made to '/login'
+    THEN verify the user
+    AND return a JWT on successful verification
+    """
+
+    # create a
+    create_user_response = client.post(
+        "/signup",
+        json={"email": "test@email.com", "username": "testUser", "password": "test"},
```

Comment:
do we/will we have min requirements for passwords? lengths, characters, etc

Reply:
This hasn't been discussed yet, length can possibly be something handled on the client side before the form data is submitted.

Reply:
i think we should consider bare minimum 8 characters, 1 symbol, 1 capital letter, 1 number?